src/main/java/org/folio/circulation/rules/Drools.java

```diff
@@ -92,6 +94,8 @@ public JsonArray loanPolicies(MultiMap params, Location location) {
       JsonObject json = new JsonObject();
 
       write(json, "loanPolicyId", match.loanPolicyId);
+      write(json, "lineNumber", match.lineNumber);
```

Comment:
What is the difference between `lineNumber` and the `circulationRuleLine` property written in the `writeLineMatch` method invoked below?

Reply:
It has been removed from the last vesion of code